Fix Wrong IRQ type for A9 global timer resulting in following warning:
 [    0.000000] GIC: PPI11 is secure or misconfigured
 [    0.000067] GIC: PPI11 is secure or misconfigured
 [    0.006356] GIC: PPI11 is secure or misconfigured

diff -aurN a/arch/arm/boot/dts/netx4000/netx4000-common.dtsi b/arch/arm/boot/dts/netx4000/netx4000-common.dtsi
--- a/arch/arm/boot/dts/netx4000/netx4000-common.dtsi	2022-03-31 07:38:56.232759949 +0200
+++ b/arch/arm/boot/dts/netx4000/netx4000-common.dtsi	2022-04-25 11:09:09.163839485 +0200
@@ -99,7 +99,7 @@
 		compatible = "arm,cortex-a9-global-timer";
 		reg = <0xfaf00200 0x20>;
 		interrupt-parent = <&gic>;
-		interrupts = <GIC_PPI (27-16) (GIC_CPU_MASK_RAW(0x1)|IRQ_TYPE_LEVEL_HIGH)>;
+		interrupts = <GIC_PPI (27-16) (GIC_CPU_MASK_RAW(0x1)|IRQ_TYPE_EDGE_RISING)>;
 		clocks = <&arm_periph_clk>;
 	};
 
